### ISSUE TITLE
Device removed

### ISSUE THREAD

**Issue by rakantoh** [x-wrt/x-wrt#408]
What happened with dlink dir 882? i can´t find the firmware for this device

**Comment by ptpt52** [comment]
Sometimes we drop support for uncommon device firmware if no one requests it. Since you requested it I can support it in the next release

**Comment by rakantoh** [comment]
yes please, i have this device (dir 882) and i upgrade every week or two weeks or a montth to new upgrade.
